returnn/config.py

```diff
@@ -160,25 +186,33 @@ def __parse_python(self, code, name=None):
         raise RuntimeError("Could not serialize %s" % code)
 
     def hash(self):
-        h = {"returnn_config": self.config, "extra_python_hash": self.extra_python_hash}
+        h = {"returnn_config": self.config, "extra_python_hash": self.python_epilog_hash}
```

Comment:
here too

Reply:
I should get a coffee... yes obviously this needs to be changed.